packages/storybook/.storybook/preview.tsx

```diff
@@ -16,11 +16,14 @@ import { THEMES } from '../stories/themes';
 import { LOCALES } from '../stories/locales'
 import { TOC } from './TOC';
 
+
```

Comment:
nit: no need for extra empty lines, just one to separate blocks is enough :)

Reply:
oh oops! I'll clear that 😊

Reply:
no worries :)